### PR TITLE
i added chrome driver support-ish it probably doesn't work well(or at all)

### PR DESCRIPTION
**Pull request by Noah-It-All**
i added the chromedriver.exe its janky and needs work but, you probably could improve it

### CODE REVIEW

File: TNAPI/login.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import selenium
+from selenium import webdriver
+from selenium.webdriver import Chrome
 from selenium.webdriver import Firefox
```
Comment (alecgirman) [inline TNAPI/login.py:4]:
```suggestion
from selenium.webdriver import Chrome, Firefox
```

File: TNAPI/login.py [hunk 2]
```diff
@@ -30,4 +34,29 @@ def login(user, password, path=r"geckodriver.exe"):
     time.sleep(3)
     sid = browser.get_cookie("connect.sid")["value"]
     browser.close()
-    return sid
\ No newline at end of file
+    return sid
+else:
```
Comment (alecgirman) [inline TNAPI/login.py:38]: Else is not used as an alternative when try fails.  See other suggestions for how it should be done.

File: TNAPI/login.py [hunk 3]
```diff
@@ -7,7 +10,8 @@
 import time
```
Comment (alecgirman) [inline TNAPI/login.py:11]:
```suggestion
webdriver = 'firefox' # use firefox by default, but can also be set to 'chrome'
```

File: TNAPI/login.py [hunk 4]
```diff
@@ -7,7 +10,8 @@
...
 
 #Functions
-def login(user, password, path=r"geckodriver.exe"):
+try:
```
Comment (alecgirman) [inline TNAPI/login.py:13]:
```suggestion
if webdriver == 'firefox':
```

File: TNAPI/login.py (hunk 2 above)
Comment (alecgirman) [inline TNAPI/login.py:38]:
```suggestion
elif webdriver == 'chrome':
```

**Review by alecgirman — CHANGES_REQUESTED** [review]

File: TNAPI/login.py (hunk 2 above)
Comment (leogomezz4t) [reply TNAPI/login.py:38]: Great thanks I'll do that

**Comment by Noah-It-All** [comment]
I have like no idea what im doing so thanks for the help @leogomezz4t im just going to commit everything you suggested

**Pull request merged**